bower.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "purescript-easyimage",
+  "homepage": "https://github.com/slamdata/purescript-easyimage",
+  "authors": [
+    "Maxim Zimaliev <zimaliev@gmail.com>"
+  ],
+  "license": "Apache-2.0",
+  "dependencies": {
+    "purescript-aff": "^0.11.3"
+  }
+}
```

Comment:
Can you add a `repository` field to the `bower.json` so we can publish on pursuit?